python/pyspark/sql/tests/test_chunk_read_api.py

```diff
@@ -0,0 +1,122 @@
+#
```

Comment:
Can you add these files into `modules.py` at `dev/sparktestsupport`? Tests are not running in CI now.